package.json

```diff
@@ -8,14 +8,17 @@
     "build": "gulp",
     "build:dist": "gulp --dist",
     "start": "nodemon dev/server/server.js",
-    "server:dist": "node dist/server/server.js"
+    "server:dist": "node dist/server/server.js",
+    "test": "./node_modules/.bin/mocha --compilers js:babel-register --bail --reporter nyan ./src/specs/**/*.js"
```

Comment:
Will it run the local version if not installed globally if just `mocha` is used? I'm not sure how this works with npm scripts.

On a side note:
I personally using explicit dev dependencies for each project (instead of global for each team member) if possible. That way different team members don't have different issues with different versions of the dev tools.